src/main/java/uk/gov/companieshouse/pscdataapi/controller/CompanyPscController.java

```diff
@@ -37,7 +38,7 @@ public ResponseEntity<Void> submitPscData(@RequestHeader("x-request-id") String
         try {
             LOGGER.info("Payload received, inserting record...");
             pscService.insertPscRecord(contextId, request);
-            return ResponseEntity.ok().build();
+            return ResponseEntity.status(HttpStatus.CREATED).build();
```

Comment:
Should there not be a unit test for this. Similar to in [statements](https://github.com/companieshouse/psc-statement-data-api/blob/main/src/test/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementControllerTest.java)